src/core/reference/include/openvino/reference/identity.hpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cstring>
+
+namespace ov {
+namespace reference {
+/**
+ * @brief Identity operation computes the identity of the input tensor.
+ *
+ * @param input Input matrix (matrices) pointer.
+ * @param output Output matrix (matrices) pointer.
```

Comment:
Detail: all parameters should be documented.

Reply:
Applied in Ref/Core!